packages/frontend/amp/components/Footer.tsx

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import { css } from 'emotion';
-import { textSans } from '@guardian/pasteup/typography';
+import { textSans, body } from '@guardian/pasteup/typography';
 import { palette } from '@guardian/pasteup/palette';
 import { InnerContainer } from './InnerContainer';
 import { Link, footerLinksNew } from '@frontend/lib/footer-links';
+import { ReaderRevenueButton } from '@root/packages/frontend/amp/components/ReaderRevenueButton';
 
 const footer = css`
     background-color: ${palette.brand.main};
     color: ${palette.neutral[86]};
-    ${textSans(5)};
+    ${textSans(3)};
```

Comment:
Is this to match the existing AMP better? I should have communicated this better, but we're aiming to mirror mobile fonts now - after chatting with @zeftilldeath on this. As existing AMP is quite out of date.

Reply:
(This applies to all of the font styles we use on new AMP.)

Reply:
Possibly yeah - will take a look!